src/main/java/com/example/demoadminpanel/transaction/model/TransactionBean.java

```diff
@@ -51,26 +53,26 @@ public static TransactionBean transformFromTransactionBeanWithUserDetails(Transa
     public static String getPaymentMethod(Integer statusId) {
```

Comment:
For future case, it could be nicer to return the enum value itself and let frontend handle the description itself (for easier language switching support).

Reply:
The issues related to enum handling (this one and the one below) will be addressed in separate requests after communication with the frontend team about the changes.